.eslintrc

```diff
@@ -0,0 +1,194 @@
+{
+    "parser": "babel-eslint",
+
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true
+    },
+
+    "ecmaFeatures": {
+        "arrowFunctions": true,
+        "binaryLiterals": true,
+        "blockBindings": true,
+        "classes": false,
+        "defaultParams": true,
+        "destructuring": true,
+        "forOf": true,
+        "generators": true,
+        "modules": true,
+        "objectLiteralComputedProperties": true,
+        "objectLiteralDuplicateProperties": true,
+        "objectLiteralShorthandMethods": true,
+        "objectLiteralShorthandProperties": true,
+        "octalLiterals": true,
+        "regexUFlag": true,
+        "regexYFlag": true,
+        "spread": true,
+        "superInFunctions": false,
+        "templateStrings": true,
+        "unicodeCodePointEscapes": true,
+        "globalReturn": true,
+        "jsx": true
+    },
+
+    "rules": {
+        "block-scoped-var": [0],
+        "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+        "camelcase": [0],
+        "comma-dangle": [2, "never"],
+        "comma-spacing": [2],
+        "comma-style": [2, "last"],
+        "complexity": [0, 11],
+        "consistent-return": [2],
+        "consistent-this": [0, "that"],
+        "curly": [2, "multi-line"],
+        "default-case": [2],
+        "dot-notation": [2, { "allowKeywords": true }],
+        "eol-last": [2],
+        "eqeqeq": [2],
+        "func-names": [0],
+        "func-style": [0, "declaration"],
+        "generator-star-spacing": [2, "after"],
+        "strict": 0,
+        "guard-for-in": [0],
+        "handle-callback-err": [0],
+        "key-spacing": [2, { "beforeColon": false, "afterColon": true }],
+        "quotes": [2, "single", "avoid-escape"],
+        "max-depth": [0, 4],
+        "max-len": [0, 80, 4],
+        "max-nested-callbacks": [0, 2],
+        "max-params": [0, 3],
+        "max-statements": [0, 10],
+        "new-parens": [2],
+        "new-cap": [0],
+        "newline-after-var": [2],
```

Comment:
Yeah, need that update there too. It actually gets bit annoying to propagate changes. I've got the same issues with my own modules. Not sure what is the best solution. Maybe some "defaults" repo with stuff and couple of scripts like `npm install && cp -f ./node_modules/defaults/.eslintrc ./`?
What do you reckon?